src/core/worker.js

```diff
@@ -197,6 +197,115 @@ IPDFStreamRangeReader.prototype = {
 }
 
 /** @implements {IPDFStream} */
+class PDFNetworkStream {
```

Comment:
Given the IRC discussion yesterday, I'm assuming that we shouldn't use `class` here (or elsewhere), since this code is in `src/core`!?